include/coc.vim

```diff
@@ -22,11 +22,7 @@ command! -nargs=0 Format :call CocAction('format')
 " Use `:Fold` for fold current buffer
 command! -nargs=? Fold :call CocAction('fold', <f-args>)
 
-
-augroup config#go#coc
-  autocmd!
-  autocmd BufWritePre *.go :CocCommand editor.action.organizeImport
-augroup END
+autocmd BufWritePre *.go :silent call CocAction('runCommand', 'editor.action.organizeImport')
```

Comment:
Is there any reason why this can't be in the `augroup`? If not, can you put in the group as to to keep the autocomands organized.

Reply:
Nope, I have no idea what augroups are for so I didn't know if that was part of the problem or just the `CocAction`/`:CocCommand` difference 😄 . Let me fix that.